picard/webservice/__init__.py

```diff
@@ -165,13 +167,13 @@ def _init_headers(self):
                 self.setRawHeader(b"Accept", self.response_mimetype.encode('utf-8'))
 
         if self.data:
-                self.setHeader(QtNetwork.QNetworkRequest.ContentTypeHeader, "application/x-www-form-urlencoded")
+                self.setHeader(QNetworkRequest.ContentTypeHeader, "application/x-www-form-urlencoded")
 
     def _update_authorization_header(self):
+        bearer = b""
```

Comment:
I'd probably call this variable "authorization", because "bearer" is just the specific authorization scheme used currently, but could be something else entirely

Reply:
Done in 716be80